### DISCUSSION TITLE
Связь кодов для документов, удостоверяющих личность, с соответствующими пространствами идентификаторов.

### DISCUSSION THREAD

**Discussion by Vrodvi** [fhir-ru/core#140] [General]
# 1)
В NamingSystem http://fhir-ru.zendoc.me/systems.identity-documents требуются корректировки:
- сейчас:
    {
      "type": "uri",
      "preferred": true,
      "value": "http://fhir.ru/core/systems/identity-documents",
    }
- в этом месте должен быть url справочника - т.е. CodeSystem, и он должен быть назван по регламенту.
Если смотреть на предыдущей странице (http://fhir-ru.zendoc.me/systems), то там url указан другой, он ближе, но тоже не соответствует регламенту. Плюс в этом url не обозначен источник справочника - то что это справочник НСИ. Мне кажется, что сторонние справочники следует обозначать с префиксом источника этого справочника.
  - должно быть:
    {
      "type": "uri",
      "preferred": true,
      "value": "http://fhir.ru/core/cs/core-cs-nsi-identity-documents",
    }	

# 2)
Для каждого типа документа, удостоверяющего личность, должно иметься пространство идентификаторов. 
- Например, такое пространство идентификаторов уже заведено для "Паспорта РФ" -
http://fhir-ru.zendoc.me/systems.passport-rf
- На совещании обсуждено, что где-то в документации должна быть обозначена связь между кодами по справочнику документов, удостоверяющего личность, с соответствующими пространствами идентификаторов.
- Озвучено, что данная связь может быть выражена формально в виде определения "срезов" в профиле, но данный момент срезы в профиле пациента не имеют этого соответствия.
- Также обсуждено, что помимо срезов в профиле желательно иметь человекочитаемый формат обозначающий соответствие кодов документов, удостоверяющих личность, и их пространств идентификаторов. 
  - один из вариантов - добавление в справочник НСИ дополнительной колонки с обозначением соответствующего пространства
  - другой вариант - простая таблица в рамках ZENDOC с маппированием кодов на соответствующего пространства (нужно что-то аналогичное представлению http://fhir-ru.zendoc.me/core.vs-identification-documents - но только в данном случае ConceptMap не применим, т.к. набор пространств идентификаторов для ДУЛ - это не справочник и на него не сделать маппирование)
Т.е. требуется:
2.1) Требуется добавить в профиле пациента в "срезы" по ДУЛ требование на использование подходящих кодов из справочника НСИ
2.2) Требуется добавить соответствие кодов документов, удостоверяющих личность, и их пространств идентификаторов в человекочитаемом формате.

**Comment by Vrodvi** [comment]
@eugenekogan-spb 
Это обсуждение по поводу вот этого пункта http://fhir-ru.zendoc.me/meetings.2022-04-26:
> 3. Сделать дискуссию о связи имен для NamingSystem конкретных документов, например паспорта (сейчас core/systems/passport-rf), и имени справочника документов удостоверяющих личность (core/systems/identity-documents) - Родионов.

**Comment by eugenekogan-spb** [comment]
### 2)
Сейчас сделан вариант 2 - "простая таблица в формате zendoc". Но есть вопрос о составе этих пространств имен. Нужно ли делать названия для всех типов документов, поименованных в справочнике "Документы, удостоверяющие личность" https://nsi.rosminzdrav.ru/#!/refbook/1.2.643.5.1.13.13.99.2.48?
В чате Николай Бельмецев - https://t.me/BelmetsevNF - предлагает ввести названия для всех документов:
https://docs.google.com/spreadsheets/d/12pwowkNrFLm7lRtFaVYGNEcnpb3lnXq_/edit?usp=sharing&ouid=107253868441487824494&rtpof=true&sd=true
